spec/support/cocina_matchers.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rspec/expectations'
+
+RSpec::Matchers.define :equal_cocina do |expected|
+  match do |actual|
+    actual.to_json == expected.to_json
+  rescue NoMethodError
+    warn "Could not match cocina models because expected is not a valid JSON string: #{expected}"
+    false
+  end
+
+  failure_message do |actual|
+    SuperDiff::EqualityMatchers::Hash.new(
+      expected: expected.to_h.deep_symbolize_keys,
+      actual: actual.to_h.deep_symbolize_keys
+    ).fail
+  rescue StandardError => e
+    "Error in CocinaMatchers: #{e}"
+  end
+end
```

Comment:
Open to using the matcher we get for free from cocina-models? https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/rspec/matchers.rb#L46-L62

Reply:
That matcher takes a JSON string and a cocina object. I've always found that confusing and isn't a good fit for tests like these where we are comparing 2 cocina objects.

That being said, this is a slight preference so happy to defer to your judgment.